docker-compose-full-services.yml

```diff
@@ -58,20 +56,21 @@ services:
     environment:
       - MIX_ENV=dev
       - ELIXIR_SERVICE=WATCHER
-      - CONTRACT_EXCHANGER_URL=http://localhost:5000
-      - ETHEREUM_RPC_URL=http://localhost:8545
-      - CHILD_CHAIN_URL=http://localhost:9656
+      - CONTRACT_EXCHANGER_URL=http://contractexchanger:5000
+      - ETHEREUM_RPC_URL=http://geth:8545
```

Comment:
this is missing `ETHEREUM_WS_RPC_URL`

Reply:
added, thanks.

Reply:
Note: this env var existed in the watcher compose file, not in full services, so i added it to the watcher file.

If we need to add it to the full services file, what should the value/port number be?

Reply:
ETHEREUM_WS_RPC_URL: ws://geth:8546

Reply:
Both in watcher and child chain full services.

Reply:
done!